### PR TITLE
Simplify installation instructions in README.

### PR DESCRIPTION
**Pull request by Zimmi48**
Now that they are well described on the website.

**Kind:** documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,12 +65,9 @@ environment for semi-interactive development of machine-checked proofs.
 [coqorg-badge]: https://images.microbadger.com/badges/version/coqorg/coq.svg
 [coqorg-link]: https://github.com/coq-community/docker-coq/wiki#docker-coq-images "coqorg/coq:latest"
 
-Download the pre-built packages of the [latest release][] for Windows and macOS;
-read the [help page][opam-using] on how to install Coq with OPAM;
-or refer to the [`INSTALL.md`](INSTALL.md) file for the procedure to install from source.
-
-[latest release]: https://github.com/coq/coq/releases/latest
-[opam-using]: https://coq.inria.fr/opam/www/using.html
+Users should refer to the https://coq.inria.fr/download page.
```
Comment (jfehrle) [inline README.md:68]:
```suggestion
Please see https://coq.inria.fr/download.
```

**Review by jfehrle — APPROVED** [review]
Let me know when this is ready to merge (as-is or with tweaks).

**Comment by Zimmi48** [comment]
I've applied your suggestion. This is ready.

**Comment by jfehrle** [comment]
@coqbot: merge now

**Comment by jfehrle** [comment]
Did you want to get this into 8.13.2?

**Comment by Zimmi48** [comment]
Not really interesting to backport this as this README is mainly read online.

**Pull request merged**